tests/test_settings.py

```diff
@@ -42,7 +42,7 @@ def test_defaults(self, temporary_environment):
         assert bot_settings.graph_github_backend_repo == "regro/cf-graph-countyfair"
         assert bot_settings.graph_repo_default_branch == "master"
         assert bot_settings.github_runner_debug is False
-        assert bot_settings.frac_update_upstream_versions == 0.1
+        assert bot_settings.frac_update_upstream_versions == 0.25
```

Comment:
This is wrong on main (and unrelated to this PR). We can also remove these tests completely if they are annoying.

Reply:
Woops sorry. I switched a default and forgot the tests. We can change the tests to assert the values are in a range or remove them. Sorry about this.